<?xml version="1.0"?>
<launch>
  <!-- <arg name="num_samples" default="10" /> -->
  <arg name="chain_start" default="base" />
  <arg name="chain_end" default="tool0" />
  <arg name="timeout" default="0.005" />
  <param name="robot_description" command="$(find xacro)/xacro.py '$(find ur_description)/urdf/ur5_robot.urdf.xacro' " />
	
</launch>
